xc-admin/packages/xc-admin-common/src/multisig_transaction/anchor.ts

```diff
@@ -0,0 +1,30 @@
+import { Idl } from "@coral-xyz/anchor";
+import { AccountMeta, TransactionInstruction } from "@solana/web3.js";
+
+type NamedAccounts = Record<string, AccountMeta>;
+type RemainingAccounts = AccountMeta[];
+export type AnchorAccounts = {
+  named: NamedAccounts;
+  remaining: RemainingAccounts;
+};
+
+export function resolveAccountNames(
```

Comment:
interesting that they don't support such a thing. it's good if they do :D

Reply:
The do for native anchor programs : https://github.com/coral-xyz/anchor/blob/master/ts/packages/anchor/src/coder/borsh/instruction.ts#L138
But neither Wormhole nor Pyth abide by this standard